### ISSUE TITLE
Additional (foreign ID) column not updated in Tenant table

### ISSUE THREAD

**Issue by mathijsqdrop** [stancl/tenancy#977]
### Bug description

When adding additional (foreign ID) columns, values do not get written to the corresponding column, but end up in de `data` JSON column.

### Steps to reproduce

I added the following migration to Central section 
```php
        $entryTier = Tier::whereName(Tier::ADVANCED)->firstOrFail();  // ADVANCED has ID of 3
        Schema::table('tenants', function (Blueprint $table) use ($entryTier) {
            $table->foreignIdFor(Tier::class)
                ->default($entryTier->id)
                ->constrained();

            $table->string('test_string')
                ->nullable()
                ->default(null);
        });
```

I can see the columns are added to the table:
```sql
create table qdrop.tenants
(
    id            varchar(255)                not null
        primary key,
    email         varchar(255)                not null,
    stripe_id     varchar(255)                null,
    pm_type       varchar(255)                null,
    pm_last_four  varchar(4)                  null,
    trial_ends_at timestamp                   null,
    created_at    timestamp                   null,
    updated_at    timestamp                   null,
    data          json                        null,
    tier_id       bigint unsigned default '3' not null,
    test_string   varchar(255)                null,
    constraint tenants_email_unique
        unique (email),
    constraint tenants_tier_id_foreign
        foreign key (tier_id) references qdrop.tiers (id)
)
    collate = utf8mb4_unicode_ci;

create index tenants_stripe_id_index
    on qdrop.tenants (stripe_id);
```

Run the following lines in the same migration file 
```php
        $tenants = Tenant::all();
        $enterpriseTier = Tier::whereName(Tier::ENTERPRISE)->firstOrFail(); // ENTERPRISE has ID of 4
        foreach ($tenants as $tenant) {
            $tenant->tier()->associate($enterpriseTier);
            $tenant->test_string = $tenant->id;
            $tenant->save();
        }
```

The `tier_id` and `test_string` are added to the `data` column and the original columns are not updated:
![image](https://user-images.githubusercontent.com/101415133/195851675-0faac266-f070-4914-8cde-9c58cc49bbaa.png)

### Expected behavior

I expect the `tier_id` and `test_string` to be written on the respectively columns in the table.

**Actual behaviour**
The `tier_id` and `test_string` are added to the `data` JSON.

### Laravel version

9.34.0

### stancl/tenancy version

3.6.0

**Comment by stancl** [comment]
Have you added the column to `getCustomColumns()`? https://tenancyforlaravel.com/docs/v3/tenants/#custom-columns

**Comment by mathijsqdrop** [comment]
No I did not. I was aware of the `getCustomColumns`, heck I even debugged that call in the `VirtualColumn` trait. But assumed (by the name of the function) it ment the functionality was the other way around:
Data fields present in the `getCustomColumns` function, would be stored in the `data` JSON column. 

My bad. I added both the `tier_id` and `test_string` to the `getCustomColumns` array and it works like it should.
